client/tests/test_client_page.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from client.models import Client
+from django.contrib.auth.models import User
+from client.views import client_check
+from ordered_product.models import OrderedProduct
+from unittest.mock import Mock
+
+
+@pytest.fixture
+def non_client_mock():
+    mock = Mock()
+    mock.client_account = User.objects.create_user(username='Asaf102',
+                                                   password='litza')
+    return mock
+
+
+class TestClientLogin:
+    @pytest.mark.django_db()
+    def test_success_entry(self, client, saved_client0):
+        client.force_login(user=saved_client0.client_account)
+        assert saved_client0 in Client.objects.all()
+        respone = client.get('/client/')
+        assert respone.status_code == 200
+
+    @pytest.mark.django_db()
+    def test_fail_entry(self, client, non_client_mock):
+        client.force_login(user=non_client_mock.client_account)
+        assert non_client_mock not in Client.objects.all()
+        respone = client.get('/client/')
+        assert respone.status_code != 200
+
+    @pytest.mark.django_db()
+    def test_client_check_false(self, non_client_mock):
+        assert non_client_mock not in Client.objects.all()
+        assert client_check(non_client_mock.client_account) is False
```

Comment:
Since this function either return a value or a boolean (never None), you can `assert client_check(non_client_mock.client_account)` 
(without the explicit "is False")

Reply:
I advised making it in the current way instead of `assert not client_check(...)`.
In case of regressions, the function can potentially return something that bools to `False`. I don't see a reason not to assert on the exact thing it returns rather than anything that bools to `False`.

Reply:
I did like Yftach said.